numpy/linalg/lapack_lite/lapack.f.patch

```diff
@@ -0,0 +1,56 @@
+@@ -57759,9 +57767,10 @@
+ *     Scan up each column tracking the last zero row seen.
+          ILACLR = 0
+          DO J = 1, N
++            I = M
+-            DO I = M, 1, -1
+-               IF( A(I, J).NE.ZERO ) EXIT
+-            END DO
++C           DO I = M, 1, -1
++C              IF( A(I, J).NE.ZERO ) EXIT
++C           END DO
```

Comment:
This is a pretty bad patch, that instead of actually rewriting this code in F77 style, just says "oh, we probably didn't need this anyway". The entire set of patches is rewriting this kind of loop.